File: 1980-find-unique-binary-string/1980-find-unique-binary-string.java
class Solution {
    public String findDifferentBinaryString(String[] nums) {
        StringBuilder ans = new StringBuilder();
        for(int i = 0; i<nums.length; i++) {
            char curr = nums[i].charAt(i);
            char opp = (curr == '0') ? '1' : '0';
            ans.append(opp);
        }
        
        return ans.toString();
    }
}

File: 1980-find-unique-binary-string/1980-find-unique-binary-string.cpp
class Solution {
public:
  
    string findDifferentBinaryString(vector<string>& nums) {
        string ans = "";
        for(int i = 0; i<nums.size(); i++) {
            char curr = nums[i][i];
            char opp = (curr == '0') ? '1' : '0';
            ans += opp;
        }

        return ans;
    }
};